rules/klayout/drc/testing/Makefile

```diff
@@ -31,18 +31,32 @@ test-DRC: test-DRC_switch  test-DRC_SC  test-DRC_main
 #=================================
 
 .ONESHELL:
-test-DRC_SC:
+test-DRC_SC : test-DRC_IO_SRAM test-DRC_SC7 test-DRC_SC9
+	@echo "========== DRC-DRC_SC is done =========="
+
+.ONESHELL:
+test-DRC_IO_SRAM:
+	@cd $(Testing_DIR)
+	@echo "========== DRC-IO_SRAM testing =========="
+	@ python3 run_sc_regression.py --path=sc_testcases/gf180mcu_fd_ip_sram__sram64x8m8wm1.gds 	\
```

Comment:
You probably want to wrap before the `--path` argument.

Reply:
Noted.